tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -261,9 +263,6 @@ void generateTileBitmap( Tile tile ) {
   }
 
   void addTileToCurrentTileCanvasView( Tile tile ) {
-    if( mTilesAlreadyRendered.contains( tile ) ) {
-      return;
-    }
```

Comment:
No we do not remove them later, at this point we already removed them so `mTilesAlreadyRendered` can't contain any tiles passed by `addTileToCurrentTileCanvasView`.

That was part of the previous PR.